### ISSUE TITLE
About Landing QA

### ISSUE THREAD

**Issue by vickibrown** [thinkshout/thinkshout#823]
https://thinkshout.com/about/

- [x] Increase size of names by 30% (at least 18 pixels)
- [x] Increase size of the titles by 20%
- [x] Add 50% more padding below images (above the name)
- [x] Change hover color of the names to hex #D24C0F
<img width="361" alt="Screen Shot 2020-05-05 at 1 53 30 PM" src="https://user-images.githubusercontent.com/21184833/81115177-24d15480-8ed8-11ea-9514-42527c871603.png">

**Comment by komejo** [comment]
Ready for review on https://a11y-2020--thinkshout.netlify.app/about/

**Comment by vickibrown** [comment]
@komejo few things. 
- [ ] The names and titles are looking too big now, reduce both by 15%
- [ ] Reduce the line height of the title when it wraps to two lines by 40%
- [ ] Add 30% more padding beneath the images
- [ ] make sure the hover color on the name is hex #D24C0F

**Comment by JulesKhong** [comment]
The link for each person is wrapped around an image that has the person's name as the alt, their name displayed and their title. So names are read twice in a row on a screen reader. 
Change the alt for the images to `alt=" "` and make sure it displays as empty quotes. That tells accessibility api's that the image is decorative and doesn't supply additional info. https://www.w3.org/WAI/tutorials/images/decorative/ See the example about the flower next to the flower name. Another alternative would be to give the anchor tag an aria-label="Person Name, job title", which would override the text its wrapped around.

**Comment by JulesKhong** [comment]
- [ ] In the footer, the **we'd love to hear from you link** has too low of contrast for it's size and weight. 
<img width="746" alt="Screen Shot 2020-05-28 at 4 13 59 PM" src="https://user-images.githubusercontent.com/19519395/83203465-5962b580-a0fe-11ea-9ed8-9f8d6fa5957a.png">

**Comment by vickibrown** [comment]
@JulesKhong it doesn't look like we use that block anymore, I can't find it on the live site. I did however, find this page! https://thinkshout.com/careers/ which needs some love. I'm going to assign it to myself to update. cc @juliaford more stuff.

**Comment by komejo** [comment]
The Careers CTA was lingering in the `/team/` page - I removed the CTA as it doesn't really add anything to the page. 

The oddness of getting 'caught' was do to this being old-fashioned floats! I'd updated it to flexbox for the modern times. Also adjusted the breakpoints a little and added a 5x wide for > 1600px. 

Ready for final QA: https://a11y-2020--thinkshout.netlify.app/about/

**Comment by JulesKhong** [comment]
- [ ] @komejo The update to the VoiceOver experience is great. Could you also apply the hover state theming on focus?
<img width="366" alt="Screen Shot 2020-06-01 at 1 26 23 PM" src="https://user-images.githubusercontent.com/19519395/83451299-85867b00-a40b-11ea-9708-0beb88dd746a.png">

**Comment by komejo** [comment]
Ah! Fixed.

**Comment by vickibrown** [comment]
@komejo looks good!

**Comment by vickibrown** [comment]
@komejo looks like a link got added to the paragraph here but it doesn't have a hover/focus states. https://a11y-2020--thinkshout.netlify.app/about/
Can you just have it underline hover/focus?
<img width="866" alt="Screen Shot 2020-06-04 at 10 44 04 AM" src="https://user-images.githubusercontent.com/21184833/83792753-544abc80-a650-11ea-844c-13497ceacee6.png">

**Comment by komejo** [comment]
Fixed and on stage: https://stage.thinkshout.com/about/
